build.gradle.kts

```diff
@@ -35,7 +31,7 @@ version = if (!project.hasProperty("useSnapshot") &&
 ) {
     val prefix = when (currentBranch) {
         null, "", "v1.x", "HEAD", "master", "main" -> ""
-        else -> "$currentBranch."
+        else -> "$baseVersion."
```

Comment:
I think this is still unintended? Otherwise the base version will be duplicated for code built on branches.